README.md

```diff
@@ -12,6 +12,85 @@ Currently the functionality is more than sufficient for our needs. However, ther
 
 We would like to give back to the open source community surrounding Android that has helped make iHeartRadio a success. By using the MIT license we hope to make this code as usable as possible.
 
+## Getting started
```

Comment:
Please move this section below the artifacts section. I would like to keep the more easily digestible sections closer to the top. Compatibility we can move to a CONTRIBUTING.md file so no need to move that section up higher.